crates/goose-cli/src/commands/session.rs

```diff
@@ -62,10 +62,15 @@ pub async fn build_session<'a>(
             .expect("should resolve executable to string path"),
     )
     .with_args(vec!["mcp", "developer"]);
+    let jbconfig = SystemConfig::stdio("goose").with_args(vec!["mcp", "jetbrains"]);
     agent
         .add_system(config)
         .await
         .expect("should start developer server");
+    agent
```

Comment:
similar to above - I guess this actually adds the system?

Reply:
ditto